src/main/java/org/cancogenvirusseq/muse/components/PayloadFileMapper.java

```diff
@@ -30,14 +30,19 @@
 import org.cancogenvirusseq.muse.model.SubmissionBundle;
 import org.cancogenvirusseq.muse.model.SubmissionFile;
 import org.cancogenvirusseq.muse.model.UploadRequest;
+import org.cancogenvirusseq.muse.utils.StringUtils;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Slf4j
 @Component
 public class PayloadFileMapper {
   private final String payloadJsonTemplate;
 
+  @Value("${tsv.multiTagDelimiter}")
+  private static final String multiTagDelimiter=";";
```

Comment:
Does the assigned semi-colon value get overwritten by the config file if a value is present?

I think I've seen default values applied by using the syntax:
```
@Value("${some.key:42}")
private int intWithDefaultValue;
```
but if what you have works then thats fine.

Reply:
Have made changes to ensure default values are applied.